.circleci/config.yml

```diff
@@ -11,7 +11,7 @@
 
 docker_config: &docker_config
   docker:
-    - image: sas42/ds2-build-env
+    - image: sas42/ds2-build-env:2018-02-01_16
```

Comment:
Lol, by "hour" I meant something like `sas42/ds2-build-env:2018-02-01_16-51-42`.

Reply:
I will change the tag name to 2018-02-01_17-00-00 when the clock strikes 5pm today.